src/components/ProtestForm/ProtestForm.js

```diff
@@ -86,34 +86,45 @@ function ProtestForm({ initialCoords, submitCallback, defaultValues = {}, afterS
     if (!editMode && !params.streetAddress) {
       alert('אנא הזינו את כתובת ההפגנה');
       return;
-    } else {
-      try {
-        params.coords = mapCenter;
-        params.dateTimeList = dateTimeList;
-        // params.recaptchaToken = recaptchaToken;
-
-        let protest = await submitCallback(params);
-
-        if (editMode) {
-          setSubmitSuccess(true);
-          setSubmitMessage('ההפגנה נשלחה בהצלחה ותתווסף למפה בזמן הקרוב :)');
-          afterSubmitCallback();
-          return;
-        }
-
-        if (protest._document) {
-          setSubmitSuccess(true);
-          setSubmitMessage('ההפגנה נשלחה בהצלחה ותתווסף למפה בזמן הקרוב :)');
-          afterSubmitCallback();
-        } else {
-          throw new Error('protest._document was null.');
-        }
-      } catch (err) {
-        console.log('error!!', err);
+    } 
+
+    if(!isValidUrl(params.telegramLink)) {
```

Comment:
@urigat telegram/whatsapp links are optional. So need to check they are valid only if a value was entered by the user.

Reply:
Done